script/lib/create-windows-installer.js

```diff
@@ -1,32 +1,45 @@
 'use strict'
 
 const electronInstaller = require('electron-winstaller')
-const fs = require('fs-extra')
+const fs = require('fs')
 const glob = require('glob')
 const path = require('path')
 
 const CONFIG = require('../config')
 
 module.exports = (packagedAppPath) => {
-  const archSuffix = process.arch === 'ia32' ? '' : '-' + process.arch
+  // const archSuffix = process.arch === 'ia32' ? '' : '-' + process.arch
   const options = {
     appDirectory: packagedAppPath,
     authors: 'GitHub Inc.',
     iconUrl: `https://raw.githubusercontent.com/atom/atom/master/resources/app-icons/${CONFIG.channel}/atom.ico`,
     loadingGif: path.join(CONFIG.repositoryRootPath, 'resources', 'win', 'loading.gif'),
     outputDirectory: CONFIG.buildOutputPath,
     noMsi: true,
-    remoteReleases: `https://atom.io/api/updates${archSuffix}?version=${CONFIG.appMetadata.version}`,
+    // remoteReleases: `https://atom.io/api/updates${archSuffix}?version=${CONFIG.appMetadata.version}`,
+    setupExe: `AtomSetup${process.arch === 'x64' ? '-x64' : ''}.exe`,
     setupIcon: path.join(CONFIG.repositoryRootPath, 'resources', 'app-icons', CONFIG.channel, 'atom.ico')
   }
 
   const cleanUp = () => {
-    for (let nupkgPath of glob.sync(`${CONFIG.buildOutputPath}/*.nupkg`)) {
+    const releasesPath = `${CONFIG.buildOutputPath}/RELEASES`
+    if (process.arch === 'x64' && fs.existsSync(releasesPath)) {
+      fs.renameSync(releasesPath, `${releasesPath}-x64`)
+    }
+
+    for (let nupkgPath of glob.sync(`${CONFIG.buildOutputPath}/atom-*.nupkg`)) {
       if (!nupkgPath.includes(CONFIG.appMetadata.version)) {
         console.log(`Deleting downloaded nupkg for previous version at ${nupkgPath} to prevent it from being stored as an artifact`)
-        fs.removeSync(nupkgPath)
+        fs.unlinkSync(nupkgPath)
+      } else {
+        if (process.arch === 'x64') {
+          const newNupkgPath = `${CONFIG.buildOutputPath}/atom-x64${path.basename(nupkgPath).slice(4)}`
```

Comment:
What's the significance of the number `4` on this line? Could that number be extracted to a constant whose name explains the significant of the number?